dor-workflow-service.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency 'activesupport', '>= 3.2.1', '< 5'
-  gem.add_dependency 'nokogiri', '~> 1.6.0'
+  gem.add_dependency 'nokogiri', '~> 1.7'
```

Comment:
I wonder if we should leave this as `~> 1.6` so that people on 1.6 aren't forced to upgrade if they aren't ready for it.